arm-monitor/2017-03-01/swagger/activityLogAlerts_API.json

```diff
@@ -0,0 +1,451 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure Activity Log Alerts API",
+        "x-ms-code-generation-settings": {
+            "name": "MonitorManagementClient"
+        },
+        "version": "2017-03-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [
+        {
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/activityLogAlerts/{activityLogAlertName}": {
```

Comment:
Is `ActivityLogAlertResource` a tracked resource? If so we must have `patch` operation support.